docs/firefox-accounts.rst

```diff
@@ -47,70 +126,144 @@ Firefox:
 .. _instructions: https://github.com/vladikoff/fxa-dev-launcher#basic-usage-example-in-os-x
 
 
-
-===============================
 Linking to accounts.firefox.com
-===============================
+-------------------------------
 
-The macros `fxa_link_fragment` and `fxa_cta_link` are designed to help create a valid link with all the necessary query string parameters. As well as including the `data-mozillaonline-link` attribute needed for the China repack.
+The ``fxa_cta_link`` macro is designed to help create a valid *call to action* (CTA) link to https://accounts.firefox.com, with all the necessary query string parameters. This macro will also generate a valid ``data-mozillaonline-link`` attribute needed for the China repack distribution.
 
+Usage
+~~~~~
 
-Tracking Firefox Accounts referrals
------------------------------------
+To use the link in a Jinja template, first import the `fxa_cta_link` macro:
 
-When we send traffic to FxA signups, we need to send uniform parameters so measurement is straightforward.
+.. code-block:: jinja
 
-All query string parameters need to pass the `query parameters validation
-<https://mozilla.github.io/application-services/docs/accounts/metrics.html#descriptions-of-metrics-related-query-parameters>`_ applied by the FxA site as well.
+    {% from "macros.html" import fxa_cta_link with context %}
 
-Entrypoints
-~~~~~~~~~~~
+A link can then be invoked using:
 
-For the entrypoint url parameter that we generate and send to FxA servers, please use the following standard format: domain-point_of_CTA. Examples from our existing site:
+.. code-block:: jinja
 
-```
-entrypoint=mozilla.org-globalnav,
-entrypoint=mozilla.org-fxa_benefits,
-entrypoint=mozilla.org-firefox_new_modal,
-```
+    {{ fxa_cta_link(
+        entrypoint='mozilla.org-firefox-accounts',
+        button_text=_('Create a Firefox Account')
+    }}
 
-We may add other entrypoints later.
+Configuration
+~~~~~~~~~~~~~
 
-If the link is part of an experiment we can indicate that the referral came from an experiment by adding the additional parameters `entrypoint_experiment` and `entrypoint_variation`
```

Comment:
Why did you strike this? I'm hoping we'll use these parameters going forward rather than all the entrypoint/source/campaign/content changes we made in e.g. the 68 test underway.

If it doesn't fit here anymore, maybe it fits near the bottom in a section?

Reply:
I chose to strike this out of the documentation because the macros do not currently support these parameters (we've never used them), so it seems a bit misleading to include them in the docs just yet. I'd suggest we file a new issue to implement support for these params, and then update the docs accordingly once developers can use them.

Reply:
I filed https://github.com/mozilla/bedrock/issues/7589 to implement these parameters and to document them 👍